templates/login.html

```diff
@@ -1,13 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Are you sure to have removed those templates? They seem to be missing in this task branch. Sure to have checked that everything works properly indeed?

Reply:
Everything is working correctly.


> Are you sure to have removed those templates? They seem to be missing in this task branch. Sure to have checked that everything works properly indeed?

Reply:
👍